File: ClientApp/src/Domain/Tasks/Models/Enums/TaskType.ts
export enum TaskType {
    Task = 1,
    Bug = 2
}

export function getTaskTypeDisplayName(type: TaskType){
    switch(type){
        case TaskType.Task:
            return 'Задача'
        case TaskType.Bug:
            return 'Баг'
    }
}

File: Tasks/Domain/Tasks/Enums/TaskType.cs
namespace Tools.Enums;

public enum TaskType
{
    Task = 1,
    Bug = 2,
}

public class TaskTypeExtensions
{
    public static Boolean IsExist<T>(T type)
    {
        if (type is null) return false;

        if(Enum.IsDefined(typeof(TaskType), type)) return true;

        return false;
    }
}
